docker.Makefile

```diff
@@ -17,6 +17,10 @@ SCHEMAS_CTNR_NAME := $(BIN_NAME)-schemas-$(TAG)
 
 BUILD_ARGS=--build-arg=EXPERIMENTAL=$(EXPERIMENTAL) --build-arg=TAG=$(TAG) --build-arg=COMMIT=$(COMMIT) --build-arg=ALPINE_VERSION=$(ALPINE_VERSION)
 
+ifneq ($(GOPROXY),)
+	BUILD_ARGS = $(BUILD_ARGS) --build-arg=GOPROXY=$(GOPROXY)
```

Comment:
Not sure if it helps, but omitting the value (and the `=`, because that implicitly is equal to `=""`) will take the value from the current environment, so if `GOPROXY` is not set as an environment variable, it won't be passed as a build-arg

```
--build-arg=GOPROXY
```